### PR TITLE
Japanese translation of en/news/_posts/2020-03-31-heap-exposure-in-so…

### PR DESCRIPTION
**Pull request by mame**
…cket-cve-2020-10933.md

### CODE REVIEW

File: ja/news/_posts/2020-03-31-heap-exposure-in-socket-cve-2020-10933.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: news_post
+title: "CVE-2020-10933: socketライブラリのヒープ暴露脆弱性について"
+author: "mame"
+translator:
+date: 2020-03-31 12:00:00 +0000
+lang: ja
```
Comment (aleksandrilyin) [inline ja/news/_posts/2020-03-31-heap-exposure-in-socket-cve-2020-10933.md:7]:
```suggestion
date: 2020-03-31 12:00:00 +0000
tags: security
lang: ja
```

File: ja/news/_posts/2020-03-31-heap-exposure-in-socket-cve-2020-10933.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+socketライブラリにヒープ暴露の脆弱性が発見されました。
+この脆弱性は [CVE-2020-10933](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10933) として登録されています。
+ユーザのみなさんにはJSON gemを更新することを強くおすすめします。
```
Comment (JuanitoFatas) [inline ja/news/_posts/2020-03-31-heap-exposure-in-socket-cve-2020-10933.md:12]:
```suggestion
ユーザのみなさんにはRubyを更新することを強くおすすめします。
```

File: ja/news/_posts/2020-03-31-heap-exposure-in-socket-cve-2020-10933.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+layout: news_post
+title: "CVE-2020-10933: socketライブラリのヒープ暴露脆弱性について"
+author: "mame"
+translator:
```
Comment (JuanitoFatas) [inline ja/news/_posts/2020-03-31-heap-exposure-in-socket-cve-2020-10933.md:5]:
```suggestion
translator: "mame"
```

😂

**Comment by mame** [comment]
@aleksandrilyin @JuanitoFatas Thanks!

**Pull request merged**